dev-requirements.txt

```diff
@@ -1,5 +1,6 @@
-sphinx
+sphinx==3.1.2
```

Comment:
We should always be on the latest version, we want to know when Sphinx does a breaking change.

Reply:
We could have dependabot bump the version when new ones come out or something since this would mean that at some point there could be a PR whos tests won't pass because tests on master don't pass... in general, a test's result should always be reproducible IMO.